### ISSUE TITLE
cluster_max_pod_num setting incorrect when creating TKE

### ISSUE THREAD

**Issue by woodylic** [terraform-providers/terraform-provider-tencentcloud#228]
### Terraform Version

0.12.13

### TencentCloudProvider Version

1.25.0

### Affected Resource(s)

* tencentcloud_kubernetes_cluster

### Terraform Configuration Files

```hcl
resource tencentcloud_kubernetes_cluster tke {
  cluster_max_pod_num     = 256
  cluster_max_service_num = 1024
  // other required fields
}
```

### Steps to Reproduce

1. `terraform apply`
2. `terraform plan`

### Expected Behavior

When executing step 2, plan shows no change.

### Actual Behavior

When executing step 2, plan shows the cluster need to re-create because cluster_max_pod_num changes from 1024 to 256.

**Comment by likexian** [comment]
Yes, there is a bug, this will fix in next release.
Thank you for feedback.

**Comment by Himer** [comment]
The fixed version v1.25.2  has been sent out,  thanks!
#230
